### DISCUSSION TITLE
Mutation addProjectV2ItemById returns that the `ProjectNext` API is deprecated

### DISCUSSION THREAD

**Discussion by kreemer** [community/community#38141] [Apps, API and Webhooks] [answered]
Hi there 👋 

I've problems using the new ProjectV2 API. When I use the command like in the [docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects#adding-an-item-to-a-project), it returns that the `ProjectNext` API is deprecated:

```console
gh api graphql -f query='
  mutation {
    addProjectV2ItemById(input: {projectId: "PN_xxxxx" contentId: "I_xxxxx"}) {
      item {
        id
      }
    }
  }'
{
  "data": {
    "addProjectV2ItemById": null
  },
  "errors": [
    {
      "type": "NOT_FOUND",
      "path": [
        "addProjectV2ItemById"
      ],
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "message": "The `ProjectNext` API is deprecated in favour of the more capable `ProjectV2` API. Follow the ProjectV2 guide at https://github.blog/changelog/2022-06-23-the-new-github-issues-june-23rd-update/, to find a suitable replacement."
    }
  ]
}
```

I think i'm using the GitHub CLI wrong but I can't figured it out where the problem is.

**Comment by david-engelmann** [comment]
[This documentation breaks down the available webhooks](https://docs.github.com/en/rest/webhooks). Also, there is a comment on the addProjectV2ItemById function that might provide some clarity.

```
Note: You cannot add and update an item in the same call. You must use addProjectV2ItemById to add the item and then use updateProjectV2ItemFieldValue to update the item.
```

**Comment by kreemer** [comment, marked as answer]
Just a minute ago I did notice that the response of the call changed. The problem was, that the node id changed. It was not PN_xxxxx anymore but PVT_xxxxx
